instrumentation/opentelemetry_phoenix/README.md

```diff
@@ -22,30 +22,18 @@ may be supplied.
 ```elixir
 def deps do
   [
-    {:opentelemetry_phoenix, "~> 1.2"}
+    {:opentelemetry_phoenix, "~> 2.0.0-beta.1"}
```

Comment:
I'm not familiar with the release conventions here, but just wanted to confirm that we want to call this beta 1 as opposed to RC 1 or something like that.

Reply:
We can do rc.